docs/collections/contribute.md

```diff
@@ -1,14 +1,116 @@
 
-# Contributing Collections
+# Contribute
+
+## Creating custom blocks
+
+### Build a block
+
+Building your own custom block is simple!
+
+1. Subclass from `Block`.
+2. Add a description alongside an `Attributes` and `Example` section in the docstring.
+3. Set a `_logo_url` to point to a relevant image.
+4. Create the `pydantic.Field`s of the block with a type annotation, `default` or `default_factory`, and a short description about the field.
+6. Define the methods of the block.
+
+For example, this is how the [Secret block is implemented](https://github.com/PrefectHQ/prefect/blob/main/src/prefect/blocks/system.py#L76-L102):
+```python
+from pydantic import Field, SecretStr
+from prefect.blocks.core import Block
+
+class Secret(Block):
+    """
+    A block that represents a secret value. The value stored in this block will be obfuscated when
+    this block is logged or shown in the UI.
+
+    Attributes:
+        value: A string value that should be kept secret.
+
+    Example:
+        ```python
+        from prefect.blocks.system import Secret
+        secret_block = Secret.load("BLOCK_NAME")
+
+        # Access the stored secret
+        secret_block.get()
+        ```
+    """
+
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5uUmyGBjRejYuGTWbTxz6E/3003e1829293718b3a5d2e909643a331/image8.png?h=250"
+
+    value: SecretStr = Field(
+        default=..., description="A string value that should be kept secret."
+    )  # ... indicates it's a required field
+
+    def get(self):
+        return self.value.get_secret_value()
+```
+
+### Inheriting an existing block
```

Comment:
We haven't been super consistent about this, but I'd like to move away from the gerund form and to an infinitive form when introducing a thing to do: Build instead of Building.